### PR TITLE
Patch 6 acp installation update

### PR DESCRIPTION
**Pull request by thomas-schwaier-spryker**
## PR Description

Updated the ACP installation guide by adding all requires steps to get SCOS ACP-enabled
- Added Defintions for ACP-Enablement, ACP-Ready and ACP-Enabled
- Provided Swimmlane Diagram to explain which party (customer/Spryker) needs to take which action in ACP-Enablement process
- Gave and end-to-end overview of all steps necessary including module updates to SCOS for ACP as well as each APP usage
- Provide guideline which is the minimal required SCOS Product release version to use ACP

## Checklist
- [ x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/acp/user/intro-to-acp/acp-installation.md [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+$config[OauthClientConstants::OAUTH_GRANT_TYPE_FOR_MESSAGE_BROKER] = OauthAuth0Config::GRANT_TYPE_CLIENT_CREDENTIALS;
+$config[OauthClientConstants::OAUTH_OPTION_AUDIENCE_FOR_MESSAGE_BROKER] = 'aop-event-platform';
+
+$config[AppCatalogGuiConstants::OAUTH_PROVIDER_NAME] = OauthAuth0Config::PROVIDER_NAME;
+$config[AppCatalogGuiConstants::OAUTH_GRANT_TYPE] = OauthAuth0Config::GRANT_TYPE_CLIENT_CREDENTIALS;
+$config[AppCatalogGuiConstants::OAUTH_OPTION_AUDIENCE] = 'aop-atrs';
+
+$config[OauthClientConstants::OAUTH_PROVIDER_NAME_FOR_PAYMENT_AUTHORIZE] = OauthAuth0Config::PROVIDER_NAME;
+$config[OauthClientConstants::OAUTH_GRANT_TYPE_FOR_PAYMENT_AUTHORIZE] = OauthAuth0Config::GRANT_TYPE_CLIENT_CREDENTIALS;
+$config[OauthClientConstants::OAUTH_OPTION_AUDIENCE_FOR_PAYMENT_AUTHORIZE] = 'aop-app';
```
Comment (pushokwhite) [inline docs/acp/user/intro-to-acp/acp-installation.md:137]: `OauthClientDependencyProvider.php` description is missing.

File: docs/acp/user/intro-to-acp/acp-installation.md [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+<details open>
+<summary>General Structure</summary>
+
+```json
+ENVIRONMENT_VARIABLE_NAME_A={
+  "CONFIGURATION_KEY_A":"SOME_VALUE_A",
+  "CONFIGURATION_KEY_B":"SOME_VALUE_B"
+}
+```
+</details>
```
[the remark this answers is not in the document]
Comment (thomas-schwaier-spryker) [reply docs/acp/user/intro-to-acp/acp-installation.md:306]: Took it straight from https://spryker.atlassian.net/wiki/spaces/AOP/pages/3232694432/Tenant+instance so we have to update both. Will take care of it

File: docs/acp/user/intro-to-acp/acp-installation.md (hunk 1 above)
Comment (thomas-schwaier-spryker) [reply docs/acp/user/intro-to-acp/acp-installation.md:137]: So then this is also missing from https://spryker.atlassian.net/wiki/spaces/AOP/pages/3230695691/HowTo+upgrade+a+customer+s+project+SCOS+application+to+work+with+ACP#3.-Configure-SCOS-to-activate-the-ACP-catalog-in-the-Back-Office and needs to be updated before I can add it here, right?

Comment (pushokwhite) [reply docs/acp/user/intro-to-acp/acp-installation.md:137]: Added a missing part.

**Pull request closed without merging**